docs/userguide/transforms/general/index.md

```diff
@@ -1265,7 +1265,7 @@ and rules. (You can, however, simulate the regex behavior with transform rules
 by using a transform rule to split the conversion rules into passes.)
 
 For more details on constructing rules, see the [Transliterator Rule
-Tutorial](rules.md) .
+Tutorial](./rules.md).
```

Comment:
This only needs a small update. The change here to the link URL is equivalent to before, actually.  (Although the fix to whitespace is nice, thanks.)

It turns out that the link isn't working because the Markdown syntax for the link is interrupted by a newline in the link text.

Would you be able to delete that line break between lines 1267 and 1268?

After you make the change, because this PR is small enough, you can just amend the single commit that you have (using `git commit --amend`) followed by a force push (`git push --force`).  That will keep the "single commit" checker bot on the PR happy.

Reply:
Also, if you need git help and/or would like me to apply the change for you, let me know.

(I wasn't able to use the Github [Suggested Changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change) feature to make things slightly easier because it would touch lines that overlap with lines already changed by the PR.)